app/models/account.rb

```diff
@@ -111,6 +111,7 @@ class Account < ApplicationRecord
 
   normalizes :username, with: ->(username) { username.squish }
 
+  scope :non_internal, -> { where(arel_table[:id].gt(0)) }
```

Comment:
Maybe skip using arel at all and use something like this?
```suggestion
  scope :non_internal, -> { where(id: 1...) }
```